virttest/utils_test/libvirt.py

```diff
@@ -3671,14 +3671,19 @@ def check_cmd_output(cmd, content, err_ignore=False, session=None):
     """
     if cmd is None:
         raise exceptions.TestFail("cmd can not be None")
+    logging.debug("check_cmd_output()")
+    if not isinstance(content, str) and not isinstance(content, list):
```

Comment:
The native string (str) – text in Python 3, bytes in Python 2, it may need be cared about

Reply:
To avoid this unnecessary confusion, I remove this part. Please double check.